docs/04 Use cases/02_erc20/02_erc20.md

```diff
@@ -106,40 +95,80 @@ mkdir erc777 && cd erc777
 truffle init
 ```
 
-From here, we move into the `erc777/contracts` folder, we delete the available example `Counter.sol` and create our own `token.sol` contract file. We use the following code to mint a new ERC777 transaction that has a name that we can define.
-For demonstration purposes we name it `PrintedTokenName` and give it the abbreviation `XRP` and mint `10000` Token with 18 decimal places (as usual on ETH).
+First, let us install the Openzeppelin codebase with `npm install @openzeppelin/contracts`.
+
+As preparation to use our blockchain with truffle, we make use of the `erc777/truffle-config.js`.
+
+As network we use 
+
+```json
+  networks: {
+    // Useful for testing. The `development` name is special - truffle uses it by default
```

Comment:
Hey! I see the comment lines with red background. I think JSON doesn't have comments, maybe we can do a small trick by replacing/removing `json` setting from the snippet to fix this?

Reply:
You are right, it actually is a JavaScript file, not JSON. Thanks for the notice